bower.json

```diff
@@ -28,12 +28,14 @@
     "test",
     "tests"
   ],
-  "devDependencies": {
+  "dependencies": {
```

Comment:
Happy to keep this change if there's a good reason, but it seems like all of these dependencies (except for AngularJS itself) are only needed when developing and/or building the project (as we provide the built file as part of the repo).

Moving these from `devDependencies` to `dependencies` would mean you end up downloading all of these packages (including the Google Closure compiler) when you just want to use this library in your own project.

Looking at wiredep, it seems that it looks at the `dependencies` entry and will automatically add in all of the project's dependencies. For that to work here, the only *run time* dependency we have would be `angularjs`, with the rest being `devDependencies`.

Unless I've misunderstood something, I think this should be changed to:

```json
"dependencies": {
  "angularjs": "~1.2.25"
},
"devDependencies": {
  "everything": "else"
}
```